packages/jellyfish-api-core/__tests__/category/account/sendTokensToAddress.test.ts

```diff
@@ -0,0 +1,95 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { SelectionModeType } from '../../../src/category/account'
+
+describe('SendTokenToAddress', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await setup()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  let from: string
+
+  async function setup (): Promise<void> {
+    from = await container.call('getnewaddress')
+    await createToken(from, 'DBTC', 200)
+    await createToken(from, 'ETH', 200)
+  }
+
+  async function createToken (address: string, symbol: string, amount: number): Promise<void> {
+    const metadata = {
+      symbol,
+      name: symbol,
+      isDAT: true,
+      mintable: true,
+      tradeable: true,
+      collateralAddress: address
+    }
+    await container.waitForWalletBalanceGTE(101)
+    await container.call('createtoken', [metadata])
+    await container.generate(1)
+
+    await container.call('utxostoaccount', [{ [address]: '100@0' }])
+    await container.generate(1)
+
+    await container.call('minttokens', [`${amount.toString()}@${symbol}`])
+    await container.generate(1)
+  }
+
+  it('should create a transaction with auto select (empty source address)', async () => {
+    const transactionHex = await client.account.sendTokensToAddress({}, { [await client.wallet.getNewAddress()]: ['2@DFI'] })
```

Comment:
<img width="775" alt="Screenshot 2021-06-03 at 6 20 31 PM" src="https://user-images.githubusercontent.com/7101806/120629463-6ac06980-c498-11eb-9a06-22d86c4d7a19.png">

Could you please improve your test by checking whether the recipient address receives 2 DFI or not?

Apply to all tests.

Reply:
Implemented that but receiving an empty array.

Reply:
Did you add this line?` await container.generate(1)`
The snippet above was actually retrieved from the jellyfish codebase. You may search and take a look at it.

Reply:
I have implemented that @jingyi2811.  Thanks for outlining that.